docs/pages/zero-trust-access/zero-trust-access.mdx

```diff
@@ -1,6 +1,285 @@
 ---
 title: Teleport Zero Trust Access
 description: Provides guides for the Teleport Zero Trust Access product.
+template: "landing-page"
 ---
 
-<DocCardList />
+import LandingHero, { LandingHeroProps } from '@site/src/components/Pages/Landing/LandingHero';
+import Resources, { ResourcesProps } from "@site/src/components/Pages/Homepage/Resources";
+import UseCasesList, { UseCasesListProps } from "@site/src/components/Pages/Landing/UseCasesList";
+
+import zeroTrustAccessImg from '@version/docs/img/zero-trust-access/zta-hero.png';
+import applicationSvg from "@site/src/components/Icon/teleport-svg/application.svg";
+import linuxServersSvg from "@site/src/components/Icon/teleport-svg/linux-servers.svg";
+import databaseSvg from "@site/src/components/Icon/teleport-svg/database-access.svg";
+import kubernetesClustersSvg from "@site/src/components/Icon/teleport-svg/kubernetes-clusters.svg";
+import windowsDesktopsSvg from "@site/src/components/Icon/teleport-svg/windows-desktops.svg";
+import autoDiscoverySvg from "@site/src/components/Icon/teleport-svg/auto-discovery.svg";
+import cloudProvidersSvg from "@site/src/components/Icon/teleport-svg/cloud-providers.svg";
+import mcpAndAiSvg from "@site/src/components/Icon/teleport-svg/mcp-and-ai.svg";
+
+<LandingHero
+  title="Teleport Zero Trust Access"
+  image={zeroTrustAccessImg}
+>
+  Easy access to all your infrastructure, on a foundation of cryptographic identity and zero trust.
+
+  [Authentication and session joining](./authentication/authentication.mdx)
+
+  [Get started with role-based access control (RBAC)](./rbac-get-started/rbac-get-started.mdx)
+
+  [Export audit events to a SIEM tool](./export-audit-events/export-audit-events.mdx)
+
+</LandingHero>
+
+<UseCasesList
+  title="Popular Topics"
```

Comment:
- Some link items use sentence case, while some use title case.
- The headings in this guide should be in sentence case to be consistent with other docs pages.